src/index.tsx

```diff
@@ -26,6 +26,9 @@ ReactDOM.render(
   document.getElementById("root")
 );
 
+if ((window as any).Cypress) {
```

Comment:
Chwilę się z tym męczyłem, żeby zrobił jakoś ładniej, ale mi się nie udało. To rozwiązanie wziąłem ze [stacka](https://stackoverflow.com/questions/12709074/how-do-you-explicitly-set-a-new-property-on-window-in-typescript)